HTML/RGB Arithmetric condensed (KingBranBran).html

```diff
@@ -0,0 +1,180 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>RGB Arithmetic — Keep Talking and Nobody Explodes Module</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+    <style>
+        table {
+            margin: 1em auto;
+        }
+		
+		.directions {
+			image-rendering: optimizeSpeed;            
+			image-rendering: -moz-crisp-edges;         
+			image-rendering: -o-crisp-edges;           
+			image-rendering: -webkit-optimize-contrast;
+			image-rendering: pixelated;
+			image-rendering: optimize-contrast;        
+			-ms-interpolation-mode: nearest-neighbor;  
+
+		}
+		
+		.colortable th {
+			width: 25%;
+		}
+    </style>
+</head>
+<body>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">RGB Arithmetic but a tiny bit better</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/RGB Arithmetic.svg" class="diagram">
+                <h2>On the Subject of RGB Arithmetic but a tiny bit better</h2>
+                <p class="flavour-text">Thats a lot of colors</p>
+				
+				<table align="left">
+					<tr>
+						<th>+</th>
+						<td>Take the sum.</td>
+					</tr>
+					<tr>
+						<th>!</th>
+						<td>Take the sum, then invert.</td>
+					</tr>
+					<tr>
+						<th>X</th>
+						<td>Multiply the digits.</td>
+					</tr>
+					<tr>
+						<th>m</th>
+						<td>The first one you see in this order: - 0 +</td>
+					</tr>
+					<tr>
+						<th>M</th>
+						<td>The first one you see in this order: + 0 -</td>
+					</tr>
+					<tr>
+						<th rowspan="3">#</th>
+						<td>Two Zeros: 0</td>
+					</tr>
+					<tr>
+						<td>One Zero: -</td>
+					</tr>
+					<tr>
+						<td>Else: +</td>
+					</tr>
+					<tr>
+						<th rowspan="2">○</th>
+						<td>Take the sum.</td>
+					</tr>
+					<tr>
+						<td>One Zero? Invert.</td>
+					</tr>
+					<tr>
+						<th rowspan="3">Ø</th>
+						<td>Same: 0</td>
+					</tr>
+					<tr>
+						<td>One Zero: -</td>
+					</tr>
+					<tr>
+						<td>Else: +</td>
+					</tr>
+					<tr>
+						<th rowspan="2">◇</th>
+						<td>No Zeros: 0</td>
+					</tr>
+					<tr>
+						<td>Else: Remove a 0.</td>
+					</tr>
+				</table>
+				<img src="img/Component/order.png" align="right" style="width:40%;float:right;margin-bottom:1em;margin-top:50px;" class="directions">

```

Comment:
Move to `img/RGB Arithmetic/order.png` instead. `Component` folder are usually for module SVG image on the top right in the first page.